docs/installation/macos.md

```diff
@@ -0,0 +1,96 @@
+# Package Installation Instructions
+
+## macOS 10.12+
+
+### Installation via Homebrew (preferred)
+
+[Homebrew][brew] is the missing package manager for macOS.
+If the `brew` command is not found,
+you need to install Homebrew following [their instructions][brew].
+
+Once you've installed Homebrew, installing PowerShell is easy.
+First, install [Homebrew-Cask][cask], so you can install more packages:
+
+```sh
+brew tap caskroom/cask
+```
+
+Now, you can install PowerShell:
+
+```sh
+brew cask install powershell
+```
+
+When new versions of PowerShell are released,
+simply update Homebrew's formulae and upgrade PowerShell:
+
+```sh
```

Comment:
Do we want to add a step on how to check if there is a newer version of PowerShell with `brew cask outdated`